lib/Whateverable.pm6

```diff
@@ -108,6 +108,29 @@ multi method irc-to-me(Message $msg where .text ~~ /:i^ uptime \s* $/) {
         ~ “implementing {.language_name} {.language_version}.”
      })
 }
+#| Get the oldest commit DateTime for $*PROGRAM. Needs $*CWD to point to the whateverable git repository.
+sub term:<birthday> {
+    my $dates = run :out, ‘git’, ‘log’, ‘--follow’, ‘--date-order’, ‘--format=%aI’, $*PROGRAM.relative;
```

Comment:
I used to write run like this, but nowadays I'd do `<git log --follow --date-order --format=%aI>, $*PROGRAM.relative`.